src/com/hms/flexyosisoftconnector/OSIsoftServer.java

```diff
@@ -82,8 +81,9 @@ public static boolean RequestHTTPS(
       String FileFields,
       String FileName) {
     int res = NO_ERROR;
-    FileName = RESPONSE_DIRECTORY + FileName;
-
+    if (!FileName.equalsIgnoreCase("")) {

```

Comment:
why not just check 
```
if (FileName.length() > 0)
```

Reply:
Seems like an inefficient way to check that its not an empty string. if only we had 1.6 and could use String.isEmpty() :)

Reply:
What happens if it is empty?

Reply:
I like FileName.length() > 0 as a check better, will change to that shortly

Reply:
Changed

Reply:
Without this if statement it would break being passed "/usr/responseDir/", but with it no responses are saved on empty file names. No response files getting saved is the desired behavior on empty filename strings